generation/internal/python.go

```diff
@@ -58,13 +58,44 @@ func pythonTestJob(ls labels.LabelSet) *Job {
 	}
 }
 
+func pythonBuildJob(ls labels.LabelSet) *Job {
+
+	steps := []config.Step{
+		checkoutStep(ls[labels.DepsPython]),
+		createArtifactsDirStep,
+		{
+			Type:    config.OrbCommand,
+			Name:    "build package",
+			Command: "python/dist",
```

Comment:
Would this error out if `setup.py` is not there ? I got the feeling that it would from the [orb-doc]( https://circleci.com/developer/orbs/orb/circleci/python#commands-dist).

Reply:
Yep good call. Let's make it only build when setup.py is present.